domains/transfer-agreements/docker-environment/grafana/dashboards/prom-dashboard.json

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Why is this dashboard here?

Reply:
Think it was copy-pasted from the Auth domain

Reply:
Deleting